cool-core/src/main/java/com/nus/cool/loader/DataLoader.java

```diff
@@ -59,6 +60,9 @@ public class DataLoader {
 
     @NonNull
     private final DataWriter writer;
+
+    @NonNull
+    private final List<Integer> metaInvariantField;
```

Comment:
List<Integer>?

Reply:
The variant won't be used so I deleted it.